app/controllers/tmdb_controller.rb

```diff
@@ -104,17 +104,15 @@ def tv_series_autocomplete
   end
 
   def tv_series
-    @show_id = params[:show_id]
-    @actor_id = params[:actor_id]
-    tmdb_handler_tv_series(@show_id)
+    show_id = params[:show_id]
+    @series = tmdb_handler_tv_series(show_id)
   end
 
   def tv_season
```

Comment:
Since the `tv_season` work calls on the `tv_series` data, this work incorporates some changes to `tv_season` to accommodate. Follow-up work on `tv_season` (and related `tv_episode`) refactoring happens in subsequent PR #252 .